### ISSUE TITLE
Convert To Interpolated String Refactoring shows in invalid situation

### ISSUE THREAD

**Issue by MoamenMohamed93** [dotnet/roslyn#40413]
**netcore3**: 

**Steps to Reproduce**:

```
const string Hello = "Hello";
const string World = "World";
const string Message = Hello + " " + World;
```

A  refactoring suggestion appears in vs to convert to interpolated string. When you apply it, it produces an error _The expression being assigned to 'Program.Message' must be constant_	

```
const string Hello = "Hello";
const string World = "World";
const string Message = $"{Hello} {World}";
```

**Expected Behavior**:

The refactoring doesn't show

**Actual Behavior**:

The refactoring shows

**Comment by AdmiralSnyder** [comment]
shouldn't the refactoring offer to make it a readonly non-const, instead?

**Comment by CyrusNajmabadi** [comment]
These are now const in the latest c#.

**Comment by sharwell** [comment]
This was fixed by #40783, and later updated to account for the new constant interpolated strings in #51517
